exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -151,6 +151,7 @@ class OTLPExporterMixin(
         timeout: Backend request timeout in seconds
         compression: gRPC compression method to use
     """
+    _MAX_EXPORT_DELAY_VALUE = 64
```

Comment:
Wondering if it makes sense for this to be a constructor argument with a default value, and for the name to reflect that it's a max backoff in seconds, so something like "max_backoff_seconds"?

Reply:
Constructor argument will be convenient for local modifications, like (here with your proposed name):

`provider = TracerProvider()

exporter = OTLPSpanExporter()
exporter.max_backoff_seconds = 5

processor = BatchSpanProcessor(exporter)
provider.add_span_processor(processor)
set_tracer_provider(provider)

tracer = get_tracer(__name__)

with tracer.start_as_current_span("test_span"):
    ...`